client/src/App.js

```diff
@@ -23,6 +24,7 @@ function App() {
         {/* <Header /> */}
         <Routes>
           <Route path="/*" element={<Home />} />
+          <Route path="/setpasspage" element={<SetPasswordPage />} />
```

Comment:
put better path name

Reply:
okay sir

Reply:
is SetPassword only better
?

Reply:
also please set the background-color of set-password page to the same as login page